### ISSUE TITLE
Support Streaming Scte35 Events

### ISSUE THREAD

**Issue by vaage** [google/shaka-packager#352]
Right now Shaka Packager only supports out-of-band cues. We would like to support in-band cues. This will need to be done after the current cue synchronization work is finished.

**Comment by astroza** [comment]
Hi,

  I'm working in this by my own. Is splice injection support from mp2t format in the roadmap for this year ? I just want to know if I'm coding in vain and duplicating the effort.

**Comment by kqyang** [comment]
No. It is on our roadmap for this year. You are welcomed to work on it and send a PR to us. Let us know if you need anything from us.

**Comment by jakubvojacek** [comment]
Hello @kqyang 

is this feature still planned? Do you have any time estimate? If we were to hire some developer to work on it, could you perhaps give us some clues where and what to change? Do you think it would that be a hard job or just small changes are necessary to start supporting scte35 markers?

Thanks a lot!

**Comment by cisiwen** [comment]
same here is this feature still under roadmap or just aborted

**Comment by joeyparrish** [comment]
There is no longer a team at Google actively developing Shaka Packager.  I'm trying to revive the project by rewriting the deps and build system (see `cmake` branch), and we are also looking for volunteer maintainers to help keep the project going long-term.  For now, I am the only maintainer, and I am mostly focused on the CMake porting effort.

@jakubvojacek, @cisiwen, PRs are very much welcome!

**Comment by cisiwen** [comment]
Hi @joeyparrish 
i am not a  c/c++ developer for long time, but I am read the code and try to connect existing cues logic with scte35 detection 
hope someday i can contribute something. really appreaciate all the effort of all contributors make a very nice product and keep it open source

**Comment by acris5** [comment]
Hello everyone!
@astroza @cisiwen @jakubvojacek Have you succeded developing this feature for packager? Any news?
